ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -1242,11 +1246,33 @@ private static void addObjectToKsqlArray(final KsqlArray array, final Object val
       array.add(SqlTimeTypes.formatDate((Date) value));
     } else if (value instanceof Time) {
       array.add(SqlTimeTypes.formatTime((Time) value));
+    } else if (value instanceof byte[]) {
+      array.add(serializeVertX3CompatibleByte((byte[]) value));
     } else {
       array.add(value);
     }
   }
 
+  /**
+   * VertX 4 changed to serialize using Base64 without padding but our server
+   * still encodes the data with padding - this uses the same serializer that
+   * the server uses to serialize the data into a string. Note that this issue
+   * only affects tests because the test uses string comparison to compare the
+   * bytes (it's schemaless, so the byte[] comes back as a string from the server).
+   *
+   * @see <a href=https://github.com/eclipse-vertx/vert.x/pull/3197>vertx#3197</a>
+   */
+  private static String serializeVertX3CompatibleByte(final byte[] bytes) {
+    try {
+      // writeValueAsString by default adds quotes to both sides to make it valid
+      // JSON
+      final String escaped = ApiJsonMapper.INSTANCE.get().writeValueAsString(bytes);
+      return escaped.substring(1, escaped.length() - 1);
+    } catch (JsonProcessingException e) {
```

Comment:
It is a test. Are there tests expecting exception of type `KsqlException`? If not, exception handling is not required.